alphagov/tsuru-performance-tests
# [#99395802] Fixed nonexistent buyers frontend URLs.

# What

Many of the URLs in `dsl/dm_buyer_paths.txt` were returning 404s.

I have replaced them by URLs that work, so we don't get 404 errors while
running performance tests.
# How to Test

Have a digital marketplace buyer's frontend. Run the performance tests.

Observe that there are not 404 errors coming from buyers frontend app.
# Who to Test

Not @RichardKnop
